components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -1212,5 +1212,11 @@ void addFileToProductDocumentation(APIProductIdentifier prodcutId, Documentation
      */
     public void addProductDocumentationContent(APIProduct apiProduct, String documentationName, String text) throws APIManagementException;
 
-
+    /**
+     *
+     * @param api
```

Comment:
Shall we complete the doc comments for the method

Reply:
Resolved